ml-proto/test/labels.wast

```diff
@@ -6,12 +6,12 @@
     )
   )
 
-  (func $loop1 (result i32)
+  (func $loop1 (param $max i32) (result i32)
     (local $i i32)
-    (set_local $i (i32.const 0))
+    (set_local $i (i32.const 1))
     (loop $exit $cont
-      (set_local $i (i32.add (get_local $i) (i32.const 1)))
-      (if (i32.eq (get_local $i) (i32.const 5))
+      (set_local $i (i32.add (get_local $i) (get_local $i)))
```

Comment:
This is doubling $i on each iteration, so the expected result would be something like 2^8, no?